mpas_analysis/config.default

```diff
@@ -1316,6 +1316,10 @@ movingAverageMonths = 1
 
 # yearStrideXTicks = 1
 
+# The number of parallel tasks occupied by each timeSeriesTransport task.
+# Analysis may run faster for large meshes when this value is set to 2 or 3
+subprocessCount = 1
+
 
 [regionalTSDiagrams]
 ## options related to plotting T/S diagrams of ocean regions
```

Comment:
@xylar, should we change the default for `parallelTaskCount` from 12 to 6?

Reply:
I think the "correct" procedure is to try analysis on too many parallel tasks and drop to fewer if you run into trouble.  I'm hesitant to make the default be to run the analysis slowly because this is needed for big meshes.

Reply:
In the longer term, I am hoping to move to Parsl, which should be able to be more aware of how much memory it uses and which will allow us to use multiple nodes.  That should eliminate the need for users to worry about how many parallel tasks to use.

Reply:
Sounds good @xylar.